core/src/main/java/org/axonframework/config/EventHandlingConfiguration.java

```diff
@@ -678,19 +670,55 @@ public EventHandlingConfiguration configureListenerInvocationErrorHandler(
     }
 
     /**
-     * Configures a {@link org.axonframework.eventhandling.ListenerInvocationErrorHandler} for the
-     * {@link org.axonframework.eventhandling.EventProcessor} of the given {@code name}. This overrides the default
-     * ListenerInvocationErrorHandler configured through the {@link org.axonframework.config.Configurer}.
+     * Configures a {@link org.axonframework.eventhandling.ListenerInvocationErrorHandler} for handlers assigned to the
+     * given {@code processingGroup}. This overrides the default ListenerInvocationErrorHandler configured through the
+     * {@link org.axonframework.config.Configurer}.
      *
-     * @param name                                  The name of the event processor
+     * @param processingGroup                       The name of the processing group
      * @param listenerInvocationErrorHandlerBuilder The {@link org.axonframework.eventhandling.ListenerInvocationErrorHandler}
-     *                                              to use for the {@link org.axonframework.eventhandling.EventProcessor}
-     *                                              with the given {@code name}
+     *                                              to use for handler assigned to this group
      * @return this {@link EventHandlingConfiguration} instance for further configuration
      */
-    public EventHandlingConfiguration configureListenerInvocationErrorHandler(String name,
+    public EventHandlingConfiguration configureListenerInvocationErrorHandler(String processingGroup,
                                                                               Function<Configuration, ListenerInvocationErrorHandler> listenerInvocationErrorHandlerBuilder) {
-        listenerInvocationErrorHandlers.put(name, listenerInvocationErrorHandlerBuilder);
+        listenerInvocationErrorHandlers.put(processingGroup, listenerInvocationErrorHandlerBuilder);
+        return this;
+    }
+
+    /**
+     * Configures the Sequencing Policy to use when invoking event handlers asynchronously. The
```

Comment:
I'd add `default` in the description here, as this function adjusts the default `SequencingPolicy`. Documentation like that is more in line with the other configure functions which adjust the default of component X.